Vizzuality/half-earth-v3
# NRC miscellaneous bug-fixing

## NRC related miscellaneous bug-fixing
### Description
Some small fixes related to National Report Cards
### Testing instructions
Visit NRCs and check the PDF report. Priority legend ramp has been fixed, also HE logo added and new URL is properly displayed on the bottom right corner.
Also if you try to manually change the ISO code of the URL and enter an inexistent one the app will redirect you to the data globe (eventually when a landing page for NRCs is created the user will be redirected there. Probably some kind of toast notifications would be desiderable to inform the user about those errors)  
### Feature relevant tickets
- [ HE logo on PDF](https://vizzuality.atlassian.net/browse/HE-56)
- [ URL width on PDF](https://vizzuality.atlassian.net/browse/HE-57)
- [ Priority layer legend gradient on PDF](https://vizzuality.atlassian.net/browse/HE-58)
- [Avoid crashing on wrong iso on URL](https://vizzuality.atlassian.net/browse/HE-64)
